runtime/vm/classallocation.c

```diff
@@ -376,8 +376,11 @@ freeClassLoader(J9ClassLoader *classLoader, J9JavaVM *javaVM, J9VMThread *vmThre
 		RELEASE_CLASS_LOADER_BLOCKS_MUTEX(javaVM);
 	}
 
-	if (classLoader->jniIDs != NULL) {
-		pool_kill(classLoader->jniIDs);
+	if (NULL != classLoader->jniIDs) {
```

Comment:
dont we still need to clear them so they dont point to garbage memory?

Reply:
This isn't real - it's solely to measure the overhead of not freeing the IDs.

Reply:
ah okay